src/main/java/uk/gov/justice/hmpps/prison/service/InmateService.java

```diff
@@ -343,6 +343,7 @@ private void setAlertsFields(final InmateDetail inmate) {
             activeAlertCount.incrementAndGet();
             alertTypes.add(a.getAlertType());
         });
+        items.sort(Comparator.comparing(Alert::getAlertId));
```

Comment:
When I tested switching prisoner search to the new endpoint the index refresh job had differences on loads of records because the alerts were in a different sequence to what they used to be.

Therefore I'm planning to do a full rebuild of the indexes using the old endpoint with the alerts in the new order - so we won't publish millions of events when we switch endpoints.